### PR TITLE
Feature/show plot info

### PR DESCRIPTION
**Pull request by masafumimori**
- Add modal dialogue that shows image of tile, index, and coordinates when clicking a tile
- Remove unnecessary files (map.tsx, sample.tsx)

### CODE REVIEW

File: components/map/index.tsx [hunk 1]
```diff
@@ -166,12 +175,13 @@ const Map = ({ width, height }: MapProps) => {
     }
 
     const onClick = (ctx: CanvasRenderingContext2D, e: MouseEvent) => {
-        const GRID_SIZE = Math.sqrt(tileMap.length)
+        if (!headerHeight) return
```
Comment (lorenzopolidori) [inline components/map/index.tsx:178]: Maybe better `if (headerHeight === undefined)`? In theory, `headerHeight` could be 0.

File: components/map/tiles/tile-info-dialog.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    visible,
+    onClose,
+    tileInfo
+}: TileInfoDialogProps) => {
+    if (!tileInfo) return null
+
+    return (
+        <ModalDialog
+            visible={visible}
+            title={'Selected tile infomation'}
```
Comment (lorenzopolidori) [inline components/map/tiles/tile-info-dialog.tsx:26]: Generally, english text should go in `strings/en.js`. I suggest we use "Terraland information" for now.

File: pages/index.js [hunk 3]
```diff
@@ -1,60 +1,116 @@
 import Head from 'next/head'
-import { useState, useContext, useEffect, useCallback } from 'react'
+import { useState, useContext, useEffect, useCallback, useRef } from 'react'
 import { strings } from '../strings/en'
-import Content from '../components/content'
+// import Content from '../components/content'
```
Comment (lorenzopolidori) [inline pages/index.js:4]: I think we can remove this commented out line

File: pages/index.js [hunk 4]
```diff
@@ -1,60 +1,116 @@
...
+                <title>{strings.siteTitle}</title>
+            </Head>
+
+            {/* <Content /> */}
+            <Map
+                onSelectTile={onSelectTile}
+                width={mapSize.width}
+                height={mapSize.height}
+                headerHeight={headerRef.current?.clientHeight}
+            />
```
Comment (lorenzopolidori) [inline pages/index.js:95]: We should change the map background colour to match the header background, i.e. `$color-mine-shaft: #242424`

**Review by lorenzopolidori — CHANGES_REQUESTED** [review]

File: components/layout.js [hunk 5]
```diff
@@ -9,29 +9,30 @@ import { UserContext } from '../context/user-context'
...
                 <link rel={'icon'} href={'/favicon.ico'} />
-                <meta
-                    name={'description'}
-                    content={strings.siteDescription} />
+                <meta name={'description'} content={strings.siteDescription} />
                 <meta name={'og:title'} content={strings.siteTitle} />
-                <meta
-                    property={'og:image'}
-                    content={''} />
+                <meta property={'og:image'} content={''} />
```
[the remark this answers is not in the document]
Comment (masafumimori) [reply components/layout.js:27]: yes this was due to Prettier. And this is related to `lineWidth` option I assume

**Review by lorenzopolidori — APPROVED** [review]

**Pull request merged**